app/Actions/Slack/AddToChannel.php

```diff
@@ -13,6 +14,8 @@ class AddToChannel
     use StaticAction;
     use SlackActionTrait;
 
+    public string $queue = 'slack-rate-limited';
```

Comment:
Can we make a global-ish constant like `QUEUE_SLACK_RATE_LIMITED` and use that as the value of `$queue` in these classes? It would be silly if some classes got put in a different queue because of a typo.